addons/point_of_sale/models/pos_config.py

```diff
@@ -732,3 +744,21 @@ def setup_invoice_journal(self, company):
                 pos_config.write({'invoice_journal_id': invoice_journal_id.id})
             else:
                 pos_config.write({'module_account': False})
+
+    def get_limited_products_loading(self):
+        self.env.cr.execute("""
+            with pm as (select product_id,max(write_date) date from stock_quant group by product_id)
+            select p.id from product_product p left join product_template t on (product_tmpl_id=t.id)
+            left join pm on (p.id=pm.product_id) where p.active and t.available_in_pos
+            order by t.priority desc, t.type DESC, coalesce(pm.date,p.write_date) desc limit %s
```

Comment:
`service` products are loaded first because of the `t.type DESC`. This doesn't make sense to me. Do we really need to sort by type?